modules/multiplayer/editor/replication_editor.cpp

```diff
@@ -281,11 +281,8 @@ ReplicationEditor::ReplicationEditor() {
 	tree->set_column_custom_minimum_width(1, 100);
 	tree->set_column_title(2, TTR("Sync"));
```

Comment:
Since it's "Replication mode" in code, should this perhaps be renamed to "Replicate"?

Reply:
I like that, fixed!